### PR TITLE
Update commands in user guide

### PR DESCRIPTION
**Pull request by aldentantan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -236,10 +236,10 @@ _Details coming soon ..._
 
 | Action     | Format, Examples                                                                                                                                                                                                       |
 |------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS ecname/EMERGENCY_CONTACT_NAME [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 ecname/Lim Jun Wei t/friend t/colleague` |
+| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS ecname/EMERGENCY_CONTACT_NAME [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665` <br> `ecname/Lim Jun Wei ecphone/98765678 ecrs/Brother` <br> `dname/Sam Lim dphone/9987766 demail/samlim@hotmail.com` <br> `t/friend t/colleague` |
```
Comment (Johnwz123) [inline docs/UserGuide.md:239]: Are the emergency contact fields and doctor fields compulsory?
Could you help add them to the command format also, right before the example?

**Review by Johnwz123 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (aldentantan) [reply docs/UserGuide.md]: Thanks for pointing that out! Emergency contact and doctor fields are compulsory.

Comment (aldentantan) [reply docs/UserGuide.md]: Thanks for pointing that out! Emergency contact and doctor fields are compulsory.

**Review by saajidshaik02 — APPROVED** [review]
LGTM!

**Review by saajidshaik02 — APPROVED** [review]
LGTM!

**Review by Johnwz123 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**